src/python_bindings/examples/datasets/Workshop.csv

```diff
@@ -0,0 +1 @@
+../../../../test_input_data/Workshop.csv
```

Comment:
How does it work? Pandas parses it?..

Reply:
It's a symlink

Reply:
Looks weird. Why don't we just copy `test_input_data/Workshop.csv` here during build process as it's done for other test data?